content/reviewers_response.md

```diff
@@ -0,0 +1,642 @@
+
+**Our goal was to maximize the interpretability of the document embedding space when characterizing preprints within the bioRxiv landscape.
+Neural networks are challenging to interpret due to their non-linear activation layers.
+We empirically show this challenge by providing word clouds generated from the first principal component between our approach and doc2vec.
+Our approach teases out quantitative vs. cellular biology concepts while Doc2Vec is hard to discern. 
+Conversely, we noticed that Doc2Vec significantly improves our centroid classifier for our journal recommendation analysis.
+We discovered this improvement by first evaluating Doc2Vec against our linear approach by randomly holding out published articles from our dataset and using the rest to predict the holdout set.
+Our subsequent evaluation was comparing these two models against our test set, which consisted of using bioRxiv preprints to predict the journal that published their corresponding published counterparts.
+Overall, methods such as Doc2Vec does improve performance at the cost of interpretability.**
+
+**Word2Vec 1st Prinicpal Component WordCloud**
+![Word2vec](https://raw.githubusercontent.com/danich1/annorxiver/3bcac4ca2e9249359361525a7e5f3ae36c71d4a5/biorxiv/pca_association_experiment/output/word_pca_similarity/figure_pieces/pca_01_cossim_word_cloud.png)
+
+**Doc2Vec 1st Prinicpal Component WordCloud**
+![Doc2vec](https://raw.githubusercontent.com/danich1/annorxiver/3bcac4ca2e9249359361525a7e5f3ae36c71d4a5/biorxiv/pca_association_experiment/output/word_pca_similarity/figure_pieces/pca_01_cossim_word_cloud_doc2vec.png)
+
+**Doc2vec vs Word2vec in Journal Recommendations**
+![Doc2vec vs Word2vec Journal Recommendation](https://raw.githubusercontent.com/danich1/annorxiver/13d9789370900a1d04fb296457fd1faf2d76818a/pmc/journal_recommendation/output/figures/doc2vec_word2vec_comparison.png)
+
+
+> Further, word2vec often seems to work even better when first trained on a larger corpus before then being applied or transferred to more specialized corpora. Personally, I also made this experience when following an example tutorial provided by the creators of the package that the authors used - which too suggests starting with existing pre-trained models. While the more restricted training done by the authors might have reduced the sensitivity of their approach (… which would likely only strengthen their claims), I would be curious whether there was an additional rationale for avoiding the former strategy that might be missed by readers (e.g.: different meanings such as "abstract" that has different meanings for scientists and non-scientists?). 
+ 
+**We chose to use an unbiased approach for training our word2vec model as our goal was to preseve latent definitions contained within preprints.
```

Comment:
```suggestion
**We did consider using a pretraining step, but we were worried that then it might be hard to differentiate the effect of pretraining from the text content of bioRxiv.
Since our goal was to explore the linguistic landscape of bioRxiv itself, we elected not to do this.
We agree with the reviewer that if the goal is to enhance the predictive performance of the similarity search webserver, this would be a good step to take.
As with above, we may consider this in a future iteration of the server, but at this point we want to make sure the work continues to characterize the linguistic landscape of bioRxiv.
```